.github/workflows/cmo-make-targets.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
     - name: Ignore if change is only in jsonnetfile.lock.json
       run: |
         # Reset jsonnetfile.lock.json if no dependencies were updated
-        changedFiles=$(git diff --name-only | grep -cv 'jsonnetfile.lock.json')
+        changedFiles=$(git diff --name-only | grep -cv 'jsonnetfile.lock.json' | cat)
```

Comment:
Could you please comment that ` | cat` ignores grep exit value, otherwise it isn't obvious . 
would  `{ grep .. || true }` which seems clearer (to me) work?